herbert_core/classes/MPIFramework/@ClusterWrapper/ClusterWrapper.m

```diff
@@ -258,6 +266,10 @@
             % and complete parallel job
             if ~isempty(obj.mess_exchange_)
                 obj.mess_exchange_.finalize_all();
+                new_mess_exchange_folder = obj.mess_exchange_.next_message_folder_name;
+                if exist(new_mess_exchange_folder,'dir')==7
+                    rmdir(new_mess_exchange_folder,'s');
```

Comment:
can this throw an exception and bring down the framework wrapper?

Reply:
this is finalize procedure -- it happens when framework is shut down anyway. Especially when user types CTRL-C